src/components/20-molecules/footer-small/README.md

```diff
@@ -56,13 +56,16 @@ export default AXAFooterSmallReact;
 
 ```js
 const languages = [
-  { text: 'DE' },
-  { text: 'FR' },
-  { text: 'IT' },
-  { text: 'EN' },
+  { key: 'de', text: 'DE' },
```

Comment:
In my opinion it would be nicer to set link property with empty values.
```{ key: 'de', text: 'DE', link: '' },
  { key: 'fr', text: 'FR', link: '' },
  { key: 'it', text: 'IT', link: '' },
  { key: 'en', text: 'EN', link: '' },

Reply:
Why? That would just be useless waste, because it is not used here.